### ISSUE TITLE
Quantization seems to be doesn't produce good accuracy ? Are there additional settings I missed?

### ISSUE THREAD

**Issue by padeirocarlos** [yhhhli/BRECQ#40]
So I tried running your code on my dataset with a pre-trained ResNet50 model. I got these results

Full precision model i got accuracy of : MobileNetV2 (58.19)
Quantized model (W8A8) i got accuracy of : MobileNetV2 (12.02)
Quantized model (W6A6) i got accuracy of : MobileNetV2 (10.12)

Full precision model i got accuracy of : ResNet-50 (65.16)
Quantized model (W8A8) i got accuracy of : ResNet-50 (13.22)
Quantized model (W6A6) i got accuracy of : ResNet-50 (11.02)

![image](https://github.com/yhhhli/BRECQ/assets/47895534/9cb80876-b072-4976-98eb-7ea7f2f95c4e)
[https://github.com/yhhhli/BRECQ/blob/main/main_imagenet.py#L201C1-L229C87](url)

My accuracy however does not come nearly as close to the float model which is around 58.19% and 65.16% but after quantization
Are there additional settings I missed?

**Comment by mj0219** [comment]
I encountered the same issue when using my pretrained model on CIFAR-10 for experiments. The accuracy is only about ten percent. Have you resolved this?

**Comment by padeirocarlos** [comment]
Hi, @mj0219 sorry, for later reply ! Could you share you model after quantization? The potentially issue can be related to batch-normal quantization process! Therefore, could share your model after quantization? I want to see batch-normal quantization  result!
